platform.txt

```diff
@@ -172,3 +172,11 @@ tools.serial_upload.path.linux64={runtime.hardware.path}/tools/linux64
 tools.serial_upload.upload.params.verbose=-d
 tools.serial_upload.upload.params.quiet=n
 tools.serial_upload.upload.pattern="{path}/{cmd}" {serial.port.file} {upload.altID} {upload.usbID} "{build.path}/{build.project_name}.bin"
+
+# blackmagic upload for generic STM32
+tools.bmp_upload.cmd=arm-none-eabi-gdb
+tools.bmp_upload.path={runtime.tools.arm-none-eabi-gcc-6-2017-q2-update.path}/bin/
+tools.bmp_upload.upload.speed=230400
```

Comment:
upload speed is not used in the cmd? It should be fine to add it, no?
` -b {upload.speed} `

Reply:
It is really a USB CDCACM so it doesn't really matter. It is going to go USB speed. But you can add it without issue